### PR TITLE
doc/rados: make sentences agree in health-checks.rst

### PR DESCRIPTION
**Pull request by zdover23**
Make sentences agree at the head of each section in doc/rados/operations/health-checks.rst. The sentences were sometimes in the imperative mood and sometimes in the declarative mood.

This commit edits the first third of
doc/rados/operations/health-checks.rst.

Zac: cf. 000228

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
Note to @anthonyeleven or whoever reviews this PR: This commit includes many lines, but I haven't changed very much of the grammar. I have just rewritten a few sentences so that every section in this file will (eventually) have the same structure. This PR can be approved without much of a review.

File: doc/rados/operations/health-checks.rst [hunk 1]
```diff
@@ -545,17 +543,17 @@ of any update to Reef or to later releases.
 OSD_UNREACHABLE
 _______________
 
-Registered v1/v2 public address of one or more OSD(s) is/are out of the
-defined `public_network` subnet, which will prevent these unreachable OSDs
-from communicating with ceph clients properly.
+The registered v1/v2 public address or addresses of one or more OSD(s) is or
+are out of the defined `public_network` subnet, which prevents these
```
Comment (anthonyeleven) [inline doc/rados/operations/health-checks.rst:547]: s/out/outside/

**Comment by zdover23** [comment]
cf. https://github.com/ceph/ceph/pull/60919

**Comment by zdover23** [comment]
Note to future Zac: incorporate all of Anthony's suggestions into a future PR.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/60920 - Squid backport
https://github.com/ceph/ceph/pull/60921 - Reef backport
https://github.com/ceph/ceph/pull/60922 - Quincy backport

**Pull request merged**